.agents/skills/create-skill/SKILL.md

```
@@ -0,0 +1,273 @@
+---
+name: create-skill
+description: Scaffold a new agent skill (SKILL.md) for this project following best practices for OpenCode and Claude Code, then wire up the matching .opencode/skills/ wrapper and command.
+---
+
+## What I do
+
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)